templates/index.html

```diff
@@ -1,24 +1,22 @@
 {% extends 'base_index.html' %}
 
```

Comment:
[re](https://github.com/canonical/canonical.com/pull/1725#discussion_r2163871295): slow loading suru animation, preloading these locally seems to solve the issue, WDYT? 

```suggestion
{% block head %}
  <link rel="preload" href="/static/json/suru_light.json" as="fetch" type="application/json" crossorigin>
  <link rel="preload" href="/static/json/suru_shadow.json" as="fetch" type="application/json" crossorigin>
{% endblock head %}
```

Reply:
That's a good idea, thanks. Let me add this to the code.